src/Umbraco.Tests.Integration/Services/MacroServiceTests.cs

```diff
@@ -6,27 +6,33 @@
 using Umbraco.Core.Cache;
 using Umbraco.Core.Logging;
 using Umbraco.Core.Models;
-
-using Umbraco.Core.Persistence.Repositories;
 using Umbraco.Core.Persistence.Repositories.Implement;
 using Umbraco.Core.Scoping;
+using Umbraco.Core.Services;
+using Umbraco.Core.Services.Implement;
+using Umbraco.Tests.Common.Builders;
+using Umbraco.Tests.Common.Builders.Extensions;
+using Umbraco.Tests.Integration.Testing;
 using Umbraco.Tests.Testing;
 
-namespace Umbraco.Tests.Services
+namespace Umbraco.Tests.Integration.Services
 {
     [TestFixture]
     [Apartment(ApartmentState.STA)]
     [UmbracoTest(Database = UmbracoTestOptions.Database.NewSchemaPerTest)]
-    public class MacroServiceTests : TestWithSomeContentBase
+    public class MacroServiceTests : UmbracoIntegrationTest
     {
-        public override void CreateTestData()
+        private MacroService MacroService => (MacroService)GetRequiredService<IMacroService>();
+
+        [SetUp]
+        public override void Setup()
         {
-            base.CreateTestData();
+            base.Setup();
```

Comment:
Still thinks this should only use the [SetUp] attribute and not override the `base.Setup`

Reply:
Agreed, have changed this too.